client/state/comments/actions.js

```diff
@@ -20,6 +20,8 @@ import {
 } from '../action-types';
 import { NUMBER_OF_COMMENTS_PER_FETCH } from './constants';
 
+export const requestComment = ( { siteId, commentId } ) => ( { siteId, commentId } );
```

Comment:
i usually push as hard as I can for fsa style.

> just trying to keep with the other comment action layout?

yep! I don't know how important it is to stay consistent here though.  🤷‍♂️